kore/src/Kore/Step/Axiom/Registry.hs

```diff
@@ -205,7 +205,7 @@ ignoreEqualityRule (EqualityRule EqualityPattern { attributes })
   -- extraction of their axioms.
   | isUnit = True
   | isIdem = True
-  | Just _ <- getOverload = True
+  | Just _ <- getOverload = False
```

Comment:
This will cause the function evaluator to attempt applying the overloading axioms; is that what we want?

Reply:
I think we do.  The current approach assumes that overloads have been pushed as low as possible, so that one has the lowest sort possible for each term.  We could try to apply them manually, as we currently do for injections; however, since overloads can have multiple arguments (think lists), applying them manually might be too laborious.